projects/rawkode.academy/platform/web/package.json

```diff
@@ -18,7 +18,7 @@
     "@astrojs/react": "4.0.0",
     "@astrojs/sitemap": "3.2.1",
     "@astrojs/tailwind": "5.1.3",
-    "@astrojs/ts-plugin": "^1.10.4",
+    "@astrojs/ts-plugin": "1.10.4",
```

Comment:
Do we need to pin this to a specific patch release?

Reply:
This should not be a problem. When you run `bunx npm-check-updates --interactive` you'll get all the updates :)